packages/lib/ren-tx/src/configs/genericMint.ts

```diff
@@ -261,14 +262,22 @@ const handleMint = async (
     try {
         const minter = deposit.mint(params);
 
-        const onConfirmation = () => {
+        const onConfirmation = (_: void, receipt: TransactionReceipt) => {
             const submittedTx = {
                 sourceTxHash,
             };
-            callback({
-                type: "ACKNOWLEDGE",
-                data: submittedTx,
-            });
+            if (receipt.status == false) {
```

Comment:
Does a similar check need to be added for burning or does that work differently?

Reply:
Let me test, we might catch the reversion when we await the burn()

Reply:
Yup, we catch it, should be ok